src/transformers/models/falcon_mamba/modeling_falcon_mamba.py

```diff
@@ -131,6 +133,15 @@ def __init__(self, config: FalconMambaConfig, layer_idx: int):
         self.out_proj = nn.Linear(self.intermediate_size, self.hidden_size, bias=config.use_bias)
         self.use_bias = config.use_bias
 
+        # Triton expects to pass RMS weights even if they are non learnable, thus we need to create these weights here
+        self.register_buffer(
```

Comment:
It seems in triton you cannot compute parameter-free RMS norm, therefore we create dummy non-learnable weights that are non persistent here